packages/stems/src/components/Modal/ModalContentPages.tsx

```diff
@@ -53,6 +55,14 @@ export const ModalContentPages = ({
     polyfill: ResizeObserver
   })
 
+  // A bit of a hack, but instead of resizing the height to the result from
+  // useMeasure, clamp it to the window height minus a padding.
+  // This makes it so that when inside a "Drawer," safe areas are respected.
+  const computedHeight = Math.min(
```

Comment:
feels like there's probably a way to handle this with CSS that takes into account what the safe areas actually are, rather than guessing with the magic 100 number?

Reply:
Yes there definitely is... I couldn't figure it out without a really serious flexy refactor. But let's chat about this some.